activesupport/lib/active_support/core_ext/module/attribute_accessors.rb

```diff
@@ -107,21 +106,20 @@ def #{sym}
   #
   #   Person.new.hair_colors = [:blonde, :red] # => NoMethodError
   #
-  # Also, you can pass a block to set up the attribute with a default value.
+  # You can set a default value for the attribute.
   #
   #   module HairColors
-  #     mattr_writer :hair_colors do
-  #       [:brown, :black, :blonde, :red]
-  #     end
+  #     mattr_writer :hair_colors, default: [:brown, :black, :blonde, :red]
   #   end
   #
   #   class Person
   #     include HairColors
   #   end
   #
   #   Person.class_variable_get("@@hair_colors") # => [:brown, :black, :blonde, :red]
-  def mattr_writer(*syms)
-    options = syms.extract_options!
+  def mattr_writer(*syms, instance_writer: nil, instance_accessor: nil, default: nil)
```

Comment:
Now seems like a good time to try it. Doubt someone used an explicit `instance_writer: nil`.